app/common/notifications/notify.service.js

```diff
@@ -217,7 +218,52 @@ function Notify(_, $q, $rootScope, $translate, SliderService, ModalService, Demo
 
         return deferred.promise;
     }
+    function deleteWithInput (type, checkName) {
+        var scope = getScope();
+        var deferred = $q.defer();
+        scope.noequal = false;
```

Comment:
can we do isEqual instead? I find it more intuitive although maybe it's just me, open to discuss